ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -2564,62 +2577,64 @@
     },
     "svCM": {
         "long_name": "Sound Velocity",
-        "units": "m/s [Chen-Millero]",
+        "units": "m/s",
+        "bodc_alternative_label": "CTD_Sound_Vel",
         "standard_name": "speed_of_sound_in_sea_water",
         "sdn_parameter_urn": "SDN:P01::SVELCA01",
-        "sdn_parameter_name": "Sound velocity in the water body by computation from salinity and temperature using UNESCO algorithm",
+        "sdn_parameter_name": "Sound velocity in the water body by computation from salinity and temperature using UNESCO algorithm [Chen-Millero]",
```

Comment:
This should be still present since it corresponds to the equation used to derive the sound speed

Reply:
Agreed !

Reply:
Sounds good, I brought back all the sounds speed